dashboard/app/models/user.rb

```diff
@@ -2297,6 +2295,14 @@ def increment_section_attempts
     save! if persisted?
   end
 
+  # This value is sent to frontend in sections_controller.rb
+  def section_attempts_last_24_hours
```

Comment:
If we change line 2275 to not check for `num_section_attempts == 0`, could we add this logic to `num_section_attempts` direction (and not have a separate function)?